### PR TITLE
Add League to adopters list

### PR DESCRIPTION
**Pull request by LoganSteinberg**
**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

NONE

**What this PR does / why we need it**:

Adding description of use cases for CNCF review by request.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -7,4 +7,6 @@ This is a list of production adopters of Falco (in alphabetical order):
...
   * https://hipaa.preferral.com/01-preferral_hipaa_compliance/
 
-* [Sumo Logic](https://www.sumologic.com/) - Sumo Logic provides a SaaS based log aggregation service that provides dashboards and applications to easily identify and analyze problems in your application and infrastructure. Sumo Logic provides native integrations for many CNCF projects, such as Falco, that allows end users to easily collect Falco events and analyze Falco events on DecSecOps focused dashboards. 
\ No newline at end of file
+* [League](https://league.com/ca/) - League provides health benefits management services to help employees understand and get the most from their benefits, and employers to provide effective, efficient plans. Falco is used to monitor our deployed services on Kubernetes, protecting against malicious access to containerswhich could lead to leaks of PHI or other sensitive data. The Falco alerts are logged in Stackdriver for grouping and further analysis. In the future, we're hoping for integrations with Prometheus and AlertManaager as well.
```
Comment (leodido) [inline ADOPTERS.md]:
```suggestion
* [League](https://league.com/ca/) - League provides health benefits management services to help employees understand and get the most from their benefits, and employers to provide effective, efficient plans. Falco is used to monitor our deployed services on Kubernetes, protecting against malicious access to containers which could lead to leaks of PHI or other sensitive data. The Falco alerts are logged in Stackdriver for grouping and further analysis. In the future, we're hoping for integrations with Prometheus and AlertManaager as well.
```

**Comment by LoganSteinberg** [comment]
I'll sign the commit tomorrow morning!

File: ADOPTERS.md [hunk 2]
```diff
@@ -7,4 +7,6 @@ This is a list of production adopters of Falco (in alphabetical order):
...
   * https://hipaa.preferral.com/01-preferral_hipaa_compliance/
 
-* [Sumo Logic](https://www.sumologic.com/) - Sumo Logic provides a SaaS based log aggregation service that provides dashboards and applications to easily identify and analyze problems in your application and infrastructure. Sumo Logic provides native integrations for many CNCF projects, such as Falco, that allows end users to easily collect Falco events and analyze Falco events on DecSecOps focused dashboards. 
\ No newline at end of file
+* [League](https://league.com/ca/) - League provides health benefits management services to help employees understand and get the most from their benefits, and employers to provide effective, efficient plans. Falco is used to monitor our deployed services on Kubernetes, protecting against malicious access to containers which could lead to leaks of PHI or other sensitive data. The Falco alerts are logged in Stackdriver for grouping and further analysis. In the future, we're hoping for integrations with Prometheus and AlertManager as well.
```
Comment (jonpulsifer) [inline ADOPTERS.md]: League should be placed before Preferral as this list is in alphabetical order

**Pull request merged**